op-chain-ops/genesis/config.go

```diff
@@ -204,6 +214,15 @@ type DeployConfig struct {
 	// RequiredProtocolVersion indicates the protocol version that
 	// nodes are recommended to adopt, to stay in sync with the network.
 	RecommendedProtocolVersion params.ProtocolVersion `json:"recommendedProtocolVersion"`
+	// SecurityCouncil represents the address of the Security Council role.
+	SecurityCouncil common.Address `json:"securityCouncil"`
```

Comment:
For naming these things, what do you think about `"contractName" || "contractVariable"` as a standard, ie `l2OutputOracleChallenger` would follow this pattern. Out of scope of this PR but if we are breaking deploy config we should standardize on something sane and stick to it

Reply:
Very much on board with that and #7310 did implement that. 
When we update the deployments, we should also rename the challenger and proposer config vars.